### PR TITLE
Integrate Pravega helloWorldWriter/Reader with the Nautilus samples.

### PR DESCRIPTION
**Pull request by co-jo**
* Add pravegaVersion property as the connector versioning is not 1 to 1.
* Update standalone README to reflect authentication.
* Update dead link in project README.

### CODE REVIEW

File: standalone-examples/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Standalone Examples of Nautilus Applications
+These applications only need a running Nautilus Deployment to execute against.
+
+## Pre requisites
+1. Java 8
+2. Nautilus Deployment
+
+## Building
+
+Prior to building, you will also need to configure your `gradle.properties` according to the configuration of your cluster. The versioning scheme for your cluster can be found at `https://{nautilusMasterIP}/versions`. In particular, for the sample(s), you will need to set:
```
[the remark this answers is not in the document]
Comment (sarlaccpit) [reply standalone-examples/README.md]: I see what you're saying.   pravegaVersion and nautilusCredentials should be set here to actual versions, which should "work" yet be transient until 0.3 officially releases, at which point we can lock it to an exact version.

In the meantime, nautilusCredentials can be set here to specifically `0.6-12625.63a53f3`

and pravegaVersion to `0.3+`, would that work?

And we remove the statement about updating the versions in this file.  Meaning everybody knows one can override properties, but the versions that are set here "should work" out of the box.

File: standalone-examples/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+There is a Linux/Mac script and a Windows (.bat) script for each separate executable.
+
+## HelloPravega Example
+This example consists of two applications, a HelloWorldReader that reads from a stream and a HelloWorldWriter, that writes to a stream.  You might want to run HelloWorldWriter in one window and HelloWorldReader in another window. 
+
+### Authentication
+Authentication is enabled by default during the creation of your cluster; you must define several **ENV** variables to build the sample with authentication enabled.
+
```
Comment (sarlaccpit) [inline standalone-examples/README.md]: We should also add a sentence explaining that the user needs to have been given ACLs on the scope for this to work.

File: standalone-examples/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (co-jo) [reply standalone-examples/README.md]: I made the incorrect assumption that the versioning should match (at least on develop) to ensure compatibility. '0.1' was just an arbitrary number I meant to remove. Given that the expectation is this should not have to be modified, I will remove this whole block.

File: standalone-examples/README.md (hunk 2 above)
Comment (co-jo) [reply standalone-examples/README.md]: Good catch, will do.

File: standalone-examples/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+### HelloWorldWriter
+A simple application that shows how to write to a Pravega stream. You can run the sample via the build scripts or by a gradle task.
+
+```
+$ bin/helloWorldWriter [-scope myScope] [-name myStream] [-uri tcp://127.0.0.1:9091] [-routingKey myRK] [-message 'hello world']
+```
+
+```
+$ gradle startHelloWorldWriter -Dexec.args="-scope myScope -name myStream -uri tcp://127.0.0.1:9091 -routingKey myRK -message hello world"
```
Comment (sarlaccpit) [inline standalone-examples/README.md]: s/127.0.0.1:9091/{nautilusMasterIP}:9091/g

**Comment by sarlaccpit** [comment]
+1

Looks good to me.

**Comment by EronWright** [comment]
These are pravega samples ported to nautilus-samples repository?   It would be nice to avoid duplication and instead focus on making the pravega-samples work directly in Nautilus.

**Comment by sarlaccpit** [comment]
It is true that this addition is very largely a duplication of what exists in the pravega/samples repo.

From what I gather, 99% of the code is the same(though @co-jo had mentioned he had to update certain things from the original which perhaps hadn't been kept up to date), except for that one build file difference:

   `compile "com.emc.nautilus:pravega-credentials:${nautilusCredentials}",`

which is a convenience to grab the nautilus credentials dependency in actuality just for runtime as nothing directly instantiates `NautilusCredentials`.

And the other addition is of course the README which explains what env variable to set etc.

One could argue that:

- the readme could exist somewhere else
- the runtime dependency could just be specified as an additional classpath parameter when launching the program.

The reason why this PR is this way is just for consistency:

- original samples were taken from pravega/samples and copied/adapted modified somewhat, to work with nautilus ( anomaly, flink samples ...).    So this was to bring in a "simple hello world" example that worked out of the box with nautilus.
- it's true in this case the original code and this one are pretty much the same, except for that build line, but the thought was to have nautilus samples be a one stop shop for folks looking to find examples for flink and straight up pravega, without having to bounce between the two repos.

**Pull request merged**